third_party/glad/BUILD.gn

```diff
@@ -0,0 +1,17 @@
+# Copyright 2014 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
```

Comment:
Is this file copied from Chromium?

If not, it should use the standard Flutter engine copyright block.